Automattic/wp-calypso
# Jetpack Cloud: open checkout in the same tab

#### Changes proposed in this Pull Request

* Make upgrade buttons located at `/scan/:site`, `/backup/:site`, and `/activity-log/:site` open their target link in the same tab.

#### Notes

* ~~Backup's upgrade button points to `https://jetpack.com/upgrade/backup/?site=[site]` rather than to checkout. Should we change this?~~ Yvonne confirmed that we are not changing any link, just making upgrade buttons open their target links on the same tab.
* I'm deleting the `client/my-sites/scan/upsell.jsx` since is a duplicate of `client/my-sites/scan/scan-upsell/index.jsx`.

#### Testing instructions

* Download this PR.
* Start Jetpack Cloud (Calypso Green) with `yarn start-jetpack-cloud`.
* Make sure you have a Jetpack site that doesn't have any paid plan.
* Visit `http://jetpack.cloud.localhost:3000/scan/[site]`.
* Click on the upgrade button.
* Make sure that checkout is opened in the same tab.
* Visit `http://jetpack.cloud.localhost:3000/backup/[site]`.
* Make sure that Jetpack Backup's marketing page is opened in the same tab.
* Visit `http://jetpack.cloud.localhost:3000/activity-log/[site]`.
* Make sure that the plans page is opened in the same tab.

Fixes https://github.com/Automattic/wp-calypso/issues/51106
Related to 1164141197617539-as-1200094501156158

#### Demo
https://user-images.githubusercontent.com/3418513/113446110-80001580-93cd-11eb-8086-7c4a55ce5508.mp4
